Upgrade to 15.4.1 not working with Brew

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Just installed Ventura 13.3.1 (a), when my iTerm2 Terminal restarted I received the message:

> A new release of Oh My Posh is available: v15.4.0 → v15.4.1
> To upgrade, run: 'brew upgrade oh-my-posh'

Perform the upgrade command and received the message
> Warning: oh-my-posh 15.4.0 already installed

### Theme

Irrelevant

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

zsh

### Log output

```Shell
Version: 15.4.0

Shell: zsh (5.9)

Segments:

ConsoleTitle(true)                         -   0 ms - zsh in scot-x
session(true)                              -   0 ms -  jonhill
path(true)                                 -   0 ms -    ~
git(false)                                 -   0 ms -
node(false)                                -   0 ms -
go(false)                                  -   0 ms -
julia(false)                               -   0 ms -
python(false)                              -   0 ms -
ruby(false)                                -   0 ms -
azfunc(false)                              -   0 ms -
aws(false)                                 -   0 ms -
root(false)                                -   0 ms -
executiontime(true)                        -   0 ms -  ﮫ0ms⠀
exit(true)                                 -   0 ms -  
shell(true)                                -   0 ms -   zsh 
ytm(false)                                 -   0 ms -
battery(true)                              -  10 ms -   92
time(true)                                 -   0 ms -  20:53:42

Run duration: 15.8665ms

Cache path: /Users/scot-x/.cache/oh-my-posh

Config path: /Users/scot-x/jandedobbeleer.omp.json

Logs:

[DEBUG] 20:53:42.600 shell.go:Getenv:394 → iTerm.app
[TRACE] 20:53:42.600 shell.go:Getenv(TERM_PROGRAM) - 70.25µs
[DEBUG] 20:53:42.600 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.600 shell.go:Getenv(XDG_CACHE_HOME) - 2µs
[TRACE] 20:53:42.600 shell_unix.go:CachePath() - 42.083µs
[DEBUG] 20:53:42.601 shell.go:Getenv:394 → /Users/scot-x/jandedobbeleer.omp.json
[TRACE] 20:53:42.601 shell.go:Getenv(POSH_THEME) - 3.792µs
[DEBUG] 20:53:42.601 shell_unix.go:Platform:71 → darwin
[TRACE] 20:53:42.601 shell.go:resolveConfigPath() - 25.25µs
[TRACE] 20:53:42.601 shell.go:Init() - 462.084µs
[TRACE] 20:53:42.601 shell.go:Flags() - 42ns
[TRACE] 20:53:42.602 config.go:loadConfig() - 790.584µs
[TRACE] 20:53:42.602 shell.go:Flags() - 41ns
[DEBUG] 20:53:42.602 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.602 shell.go:Getenv(OMP_CACHE_DISABLED) - 3.625µs
[DEBUG] 20:53:42.602 shell.go:Shell:641 → process name: zsh
[TRACE] 20:53:42.602 shell.go:Shell() - 118.958µs
[DEBUG] 20:53:42.602 shell.go:Getenv:394 → 5.9
[TRACE] 20:53:42.602 shell.go:Getenv(POSH_SHELL_VERSION) - 2.333µs
[DEBUG] 20:53:42.602 debug.go:PrintDebug:22 → Segment: Title
[TRACE] 20:53:42.602 shell_unix.go:Root() - 708ns
[TRACE] 20:53:42.602 shell.go:Shell() - 84ns
[TRACE] 20:53:42.602 shell.go:ErrorCode() - 42ns
[DEBUG] 20:53:42.602 shell_unix.go:IsWsl:35 → false
[TRACE] 20:53:42.602 shell_unix.go:IsWsl() - 22.209µs
[TRACE] 20:53:42.602 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.602 shell.go:Pwd:424 → /Users/scot-x
[TRACE] 20:53:42.602 shell.go:Pwd() - 41.416µs
[TRACE] 20:53:42.602 shell.go:GOOS() - 41ns
[TRACE] 20:53:42.602 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.602 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.602 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.602 shell.go:GOOS() - 83ns
[TRACE] 20:53:42.602 shell.go:GOOS() - 0s
[TRACE] 20:53:42.602 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.602 shell.go:GOOS() - 0s
[DEBUG] 20:53:42.602 shell.go:User:545 → jonhill
[TRACE] 20:53:42.602 shell.go:User() - 2.166µs
[DEBUG] 20:53:42.602 shell.go:Host:557 → 88888888-2.fritz.box
[TRACE] 20:53:42.602 shell.go:Host() - 8.458µs
[TRACE] 20:53:42.602 shell.go:GOOS() - 41ns
[DEBUG] 20:53:42.602 shell.go:Getenv:394 → 1
[TRACE] 20:53:42.602 shell.go:Getenv(SHLVL) - 2.167µs
[TRACE] 20:53:42.602 shell.go:TemplateCache() - 167.542µs
[TRACE] 20:53:42.602 shell.go:Flags() - 42ns
[DEBUG] 20:53:42.602 block.go:Debug:230 → Segment: session
[TRACE] 20:53:42.602 shell.go:Pwd() - 125ns
[TRACE] 20:53:42.602 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.602 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.602 shell.go:Getenv(SSH_CONNECTION) - 1.875µs
[DEBUG] 20:53:42.602 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.602 shell.go:Getenv(SSH_CLIENT) - 6.959µs
[TRACE] 20:53:42.602 shell.go:TemplateCache() - 42ns
[TRACE] 20:53:42.602 shell.go:TemplateCache() - 84ns
[TRACE] 20:53:42.603 shell.go:Shell() - 42ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: path
[TRACE] 20:53:42.603 shell.go:Pwd() - 84ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.603 shell.go:Pwd() - 83ns
[TRACE] 20:53:42.603 shell.go:Shell() - 83ns
[TRACE] 20:53:42.603 shell.go:Shell() - 84ns
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 20:53:42.603 properties.go:GetString:28 → 
[DEBUG] 20:53:42.603 properties.go:GetString:28 → 
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.603 properties.go:GetString:28 → ~
[DEBUG] 20:53:42.603 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 20:53:42.603 shell.go:PathSeparator() - 0s
[TRACE] 20:53:42.603 shell.go:GOOS() - 84ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 83ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 83ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 84ns
[TRACE] 20:53:42.603 shell.go:PathSeparator() - 42ns
[TRACE] 20:53:42.603 shell.go:PathSeparator() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 41ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.603 shell.go:PathSeparator() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.603 shell.go:PathSeparator() - 83ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 0s
[TRACE] 20:53:42.603 shell.go:PathSeparator() - 83ns
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[DEBUG] 20:53:42.603 shell_unix.go:IsWsl:35 → false
[TRACE] 20:53:42.603 shell_unix.go:IsWsl() - 2.125µs
[TRACE] 20:53:42.603 shell.go:StackCount() - 41ns
[TRACE] 20:53:42.603 shell_unix.go:DirIsWritable(/Users/scot-x) - 21.083µs
[TRACE] 20:53:42.603 shell.go:TemplateCache() - 42ns
[TRACE] 20:53:42.603 shell.go:TemplateCache() - 125ns
[TRACE] 20:53:42.603 shell.go:Shell() - 83ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: git
[TRACE] 20:53:42.603 shell.go:Pwd() - 83ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.603 shell_unix.go:IsWsl:35 → false
[TRACE] 20:53:42.603 shell_unix.go:IsWsl() - 2.625µs
[TRACE] 20:53:42.603 shell_unix.go:IsWsl2() - 4.958µs
[TRACE] 20:53:42.603 shell.go:GOOS() - 83ns
[DEBUG] 20:53:42.603 shell.go:CommandPath:596 → /opt/homebrew/bin/git
[TRACE] 20:53:42.603 shell.go:CommandPath(git) - 55.417µs
[TRACE] 20:53:42.603 shell.go:HasCommand(git) - 57.667µs
[TRACE] 20:53:42.603 shell.go:Pwd() - 41ns
[ERROR] 20:53:42.603 shell.go:HasParentFilePath:729 → stat .git: no such file or directory
[TRACE] 20:53:42.603 shell.go:HasParentFilePath(.git) - 72.959µs
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: node
[TRACE] 20:53:42.603 shell.go:Pwd() - 125ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .npmrc .vue]
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → home_enabled: false
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: go
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → extensions: [*.go go.mod]
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → home_enabled: false
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: julia
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → extensions: [*.jl]
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → home_enabled: false
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: python
[DEBUG] 20:53:42.603 properties.go:GetString:28 → files
[TRACE] 20:53:42.603 shell.go:Pwd() - 41ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv env venv-win pyenv-win]
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → home_enabled: false
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: ruby
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 0s
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → extensions: [*.rb Rakefile Gemfile]
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → home_enabled: false
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: azfunc
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → extensions: [host.json local.settings.json function.json]
[DEBUG] 20:53:42.603 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → home_enabled: false
[TRACE] 20:53:42.603 shell.go:Pwd() - 41ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: aws
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 41ns
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → display_default: false
[DEBUG] 20:53:42.603 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.603 shell.go:Getenv(AWS_VAULT) - 1.834µs
[DEBUG] 20:53:42.603 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.603 shell.go:Getenv(AWS_PROFILE) - 1.5µs
[DEBUG] 20:53:42.603 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.603 shell.go:Getenv(AWS_REGION) - 1.458µs
[DEBUG] 20:53:42.603 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.603 shell.go:Getenv(AWS_DEFAULT_REGION) - 1.292µs
[DEBUG] 20:53:42.603 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.603 shell.go:Getenv(AWS_CONFIG_FILE) - 1.375µs
[DEBUG] 20:53:42.603 shell.go:FileContent:519 ↓
    [default]

[TRACE] 20:53:42.603 shell.go:FileContent(/Users/scot-x/.aws/config) - 31.333µs
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: root
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.603 shell_unix.go:Root() - 416ns
[DEBUG] 20:53:42.603 block.go:Debug:230 → Segment: executiontime
[TRACE] 20:53:42.603 shell.go:Pwd() - 42ns
[TRACE] 20:53:42.603 shell.go:GOOS() - 0s
[DEBUG] 20:53:42.603 properties.go:GetBool:22 → always_enabled: true
[TRACE] 20:53:42.603 shell.go:ExecutionTime() - 41ns
[DEBUG] 20:53:42.603 properties.go:GetFloat64:34 → threshold: 500.000000
[DEBUG] 20:53:42.603 properties.go:GetString:28 → austin
[TRACE] 20:53:42.603 shell.go:TemplateCache() - 42ns
[TRACE] 20:53:42.604 shell.go:TemplateCache() - 167ns
[TRACE] 20:53:42.604 shell.go:Shell() - 84ns
[DEBUG] 20:53:42.604 block.go:Debug:230 → Segment: exit
[TRACE] 20:53:42.604 shell.go:Pwd() - 41ns
[TRACE] 20:53:42.604 shell.go:GOOS() - 42ns
[TRACE] 20:53:42.604 shell.go:ErrorCode() - 41ns
[DEBUG] 20:53:42.604 properties.go:GetBool:22 → always_enabled: true
[TRACE] 20:53:42.604 shell.go:TemplateCache() - 0s
[TRACE] 20:53:42.604 shell.go:Shell() - 42ns
[TRACE] 20:53:42.604 shell.go:TemplateCache() - 125ns
[TRACE] 20:53:42.604 shell.go:Flags() - 83ns
[DEBUG] 20:53:42.604 block.go:Debug:230 → Segment: shell
[TRACE] 20:53:42.604 shell.go:Pwd() - 41ns
[TRACE] 20:53:42.604 shell.go:GOOS() - 41ns
[DEBUG] 20:53:42.604 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 20:53:42.604 shell.go:Shell() - 41ns
[TRACE] 20:53:42.604 shell.go:Flags() - 0s
[TRACE] 20:53:42.604 shell.go:TemplateCache() - 83ns
[TRACE] 20:53:42.605 shell.go:TemplateCache() - 250ns
[TRACE] 20:53:42.605 shell.go:Shell() - 166ns
[DEBUG] 20:53:42.605 block.go:Debug:230 → Segment: ytm
[TRACE] 20:53:42.605 shell.go:Pwd() - 167ns
[TRACE] 20:53:42.605 shell.go:GOOS() - 41ns
[DEBUG] 20:53:42.605 properties.go:GetString:28 → http://127.0.0.1:9863
[DEBUG] 20:53:42.605 properties.go:GetInt:40 → api_url: 20
[DEBUG] 20:53:42.605 shell.go:HTTPRequest:685 ↓
    GET /query HTTP/1.1
    Host: 127.0.0.1:9863
    User-Agent: Go-http-client/1.1
    Accept-Encoding: gzip


[ERROR] 20:53:42.605 shell.go:HTTPRequest:689 → Get "http://127.0.0.1:9863/query": dial tcp 127.0.0.1:9863: connect: connection refused
[TRACE] 20:53:42.605 shell.go:HTTPRequest(http://127.0.0.1:9863/query) - 520.625µs
[DEBUG] 20:53:42.605 block.go:Debug:230 → Segment: battery
[TRACE] 20:53:42.605 shell.go:Pwd() - 42ns
[TRACE] 20:53:42.605 shell.go:GOOS() - 41ns
[DEBUG] 20:53:42.605 shell_unix.go:IsWsl:35 → false
[TRACE] 20:53:42.605 shell_unix.go:IsWsl() - 2.125µs
[DEBUG] 20:53:42.615 shell.go:RunCommand:575 ↓
    Now drawing from 'Battery Power'
     -InternalBattery-0 (id=8192099)	92%; discharging; 6:17 remaining present: true
[TRACE] 20:53:42.615 shell.go:RunCommand(pmset -g batt) - 9.884167ms
[TRACE] 20:53:42.615 shell_darwin.go:BatteryState() - 9.967958ms
[DEBUG] 20:53:42.615 properties.go:GetString:28 → 
[TRACE] 20:53:42.615 shell.go:TemplateCache() - 125ns
[TRACE] 20:53:42.615 shell.go:TemplateCache() - 42ns
[TRACE] 20:53:42.616 shell.go:Shell() - 125ns
[TRACE] 20:53:42.616 shell.go:TemplateCache() - 41ns
[TRACE] 20:53:42.616 shell.go:TemplateCache() - 166ns
[DEBUG] 20:53:42.616 block.go:Debug:230 → Segment: time
[TRACE] 20:53:42.616 shell.go:Pwd() - 166ns
[TRACE] 20:53:42.616 shell.go:GOOS() - 42ns
[DEBUG] 20:53:42.616 properties.go:GetString:28 → 15:04:05
[TRACE] 20:53:42.616 shell.go:TemplateCache() - 41ns
[TRACE] 20:53:42.616 shell.go:TemplateCache() - 42ns
[TRACE] 20:53:42.616 shell.go:Shell() - 84ns
[DEBUG] 20:53:42.616 shell.go:Getenv:394 → NO DATA
[TRACE] 20:53:42.616 shell.go:Getenv(XDG_CACHE_HOME) - 5.875µs
[TRACE] 20:53:42.616 shell_unix.go:CachePath() - 28.708µs
[TRACE] 20:53:42.616 shell.go:Flags() - 41ns
 jonhill    ~  ﮫ43ms⠀                     zsh   92  20:53:42 
```

--- a/src/upgrade/notice.go
+++ b/src/upgrade/notice.go
@@ -36,8 +36,8 @@ A new release of Oh My Posh is available: %s → %s
 `
 	windows = `To upgrade, use the guide for your favorite package manager in the documentation:
 https://ohmyposh.dev/docs/installation/windows#update`
-	unix   = "To upgrade, use your favorite package manager or, if you used Homebrew to install, run: 'brew upgrade oh-my-posh'"
-	darwin = "To upgrade, run: 'brew upgrade oh-my-posh'"
+	unix   = "To upgrade, use your favorite package manager or, if you used Homebrew to install, run: 'brew update && brew upgrade oh-my-posh'"
+	darwin = "To upgrade, run: 'brew update && brew upgrade oh-my-posh'"
 
 	CACHEKEY = "upgrade_check"
 )

